src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,277 @@
+                       type=str,
+                       nargs='+',
+                       help='List of all files if they do not '
+                       'have same file prefix. You must include \'.h5\' '
+                       'extension ')
+    parser.add_argument('--var',
+                        type=str,
+                        required=True,
+                        help='Name of variable to render. E.g. \'psi '
+                        'psi-Error\' or \'psi-Analytic\'')
+    parser.add_argument(
+        '--time',
+        type=int,
+        required=False,
+        help='If specified renders the integer observation step'
+        ' instead of an animation (optional)')
+    parser.add_argument('-o',
+                        '--output',
+                        type=str,
+                        required=False,
+                        help='Set to the name of the output file you want '
+                        'written. For animations this saves an mp4 file and '
+                        'for stills a pdf. Name of file should not include '
+                        'file extension. If not specified, renders to screen. '
+                        '(optional)')
+    parser.add_argument(
+        '--fps',
+        type=int,
+        required=False,
+        default=30,
+        help='Set the number of frames per second when writing '
+        'an animation to disk.(optional)')
+    parser.add_argument(
+        '--interval',
+        type=int,
+        required=False,
+        default=200,
+        help='Delay between frames in  milliseconds (optional)')
+    parser.add_argument('--list-vars',
+                        type=bool,
+                        required=False,
+                        help='If true then print to screen vars in h5 file '
+                        'and exit (optional)')
+    # Check for h5 suffix if '--file-prefix' argument used
+    if parser.parse_args().file_prefix is not None and \
+       parser.parse_args().file_prefix.endswith('.h5'):
+        print("Retry without .h5 extension on 'file-prefix'")
+        sys.exit(1)
+
+    return vars(parser.parse_args())
+
+
+def get_h5_files(file_prefix, filename_list):
+    '''
+    Get a list of the h5 files containing the data
+    :return list of h5 files containing volume data
+    '''
+    h5_files = []
+    if file_prefix is not None:
+        h5_file_names = glob.glob(file_prefix + "*.h5")
+        if not h5_file_names:
+            sys.exit("Could not find files with prefix %s in path" %
+                     file_prefix)
+        h5_files = [h5py.File(file_name, 'r') for file_name in h5_file_names]
+
+    else:
+        for file_name in filename_list:
+            try:
+                h5_files.append(h5py.File(file_name, 'r'))
+            except OSError:
+                exit("Could not find file %s in directory %s" %
+                     (file_name, os.path.dirname(file_name)))
+    return h5_files
+
+
+def print_var_names(args):
```

Comment:
Take only the argument needed by this function:

```suggestion
def print_var_names(files):
```

Then pass it on to `get_h5_files`.